src/index.css

```diff
@@ -1,3 +1,8 @@
+@import "styles/header.css";
```

Comment:
Having centralized css-directory is a good approach, however it ruins the concept of react component re-usability. THe best way to store css files is near the component js (jsx) file itself, stored in a separate component folder.